pkg/cloud/azure/services/virtualmachines/virtualmachines.go

```diff
@@ -120,7 +120,13 @@ func (s *Service) CreateOrUpdate(ctx context.Context, spec azure.Spec) error {
 		AdminPassword: to.StringPtr(randomPassword),
 	}
 
-	if sshKeyData != "" {
+	if compute.OperatingSystemTypes(vmSpec.OSDisk.OSType) == compute.Windows {
+		osProfile.WindowsConfiguration = &compute.WindowsConfiguration{
+			ProvisionVMAgent:       to.BoolPtr(true),
+			EnableAutomaticUpdates: to.BoolPtr(false),
+			TimeZone:               to.StringPtr("Central Standard Time"),
```

Comment:
Is it safe to hard code this? Customers could bring up clusters in other regions which should result in mapping to other time zones.

Reply:
Since we don't set the timezone anywhere (ie on Linux) at the moment, I would suggest we just leave it to default this value if we can. Is it a required field? If so,  I would suggest going for UTC rather than a specific timezone as UTC is fairly standard for servers to use as their timezone

Reply:
AFAIK it is a required field. But yes, UTC is a safer option if going with a default value.

Reply:
It is not only safer, but the only allowed one. If we set any other at this point, it will return back to `UTC` after vm restart, and may cause issues: https://techcommunity.microsoft.com/t5/azure-database-support-blog/lesson-learned-4-modifying-the-default-time-zone-for-your-local/ba-p/368798 @enxebre @JoelSpeed @aravindhp

Reply:
Good find! Sounds good to me then

Reply:
>AFAIK it is a required field

How did we conclude that it needs to set explicitly? Also any particular reason to set ProvisionVMAgent and EnableAutomaticUpdates?

Reply:
It does not have a default value, but the `ProvisionVMAgent` does. Will put a small fix. https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.management.compute.models.windowsconfiguration?view=azure-dotnet#properties

Reply:
Please don't set `EnableAutomaticUpdates` to true as we do not want automatic Windows OS updates at least for the first release of the product.

Reply:
I'm reading https://techcommunity.microsoft.com/t5/azure-database-support-blog/lesson-learned-4-modifying-the-default-time-zone-for-your-local/ba-p/368798 differently -> "All Azure services use UTC time zone settings, regardless of their physical location. " I'm reading that as UTC is the default.

Can we validate we need to set any of this fields at all?

Reply:
Removed all that not needed, left default `UTC` for explanation.